config/database.yml

```diff
@@ -29,10 +29,10 @@ development:
   # To create additional roles in postgres see `$ createuser --help`.
   # When left blank, postgres will use the default role. This is
   # the same name as the operating system user that initialized the database.
-  username: Admin
+  username: <%= ENV['DATABASE_USER'] || 'postgres' %>
```

Comment:
update README file with information the environment variables

Reply:
Information added to the README

Reply:
Information added to the README